### ISSUE TITLE
Notebook adaptation for TF v1.0.0-alpha

### ISSUE THREAD

**Issue by ddtm** [poolio/unrolled_gan#1]
One has to modify the notebook in several places:

- `sample_n(num)` calls should be replaced with `sample(num)`
- `tf.nn.sigmoid_cross_entropy_with_logits` must be supplied with explicit keyword arguments `logits` and `labels`.

**Comment by poolio** [comment]
Thanks! These are now fixed in the latest notebook.
